src/v2/configs/tenant/YJSG-Indore/strings.json

```
@@ -288,5 +288,7 @@
 	"ACTIVE": "Active",
 	"INACTIVE": "Inactive",
 	"SEE_ALL": "See All",
-	"SEE_LESS": "See Less"
+	"SEE_LESS": "See Less",
+	"COORDINATOR_NAME": "Coordinator name: ",
```

Do not include symbols on strings.json.
```suggestion
	"COORDINATOR_NAME": "Coordinator name",
```